index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="./css/styles.css">
+    <script src="https://kit.fontawesome.com/cebf050324.js" crossorigin="anonymous"></script>
+  </head>
+  <body>
+    <div class="background">
+      <nav>
+        <ul class="list-format font-color">
+          <li><a href="#">Portfolio</a></li>
+          <li><a href="#">About</a></li>
+          <li><a href="#">Contact</a></li>
+        </ul>
+      </nav>
+      <header>
+          <h1>Howdy! I am Flover. <span> I am a software developer</span></h1>
+          <p class="font-color">
+             I can assist you building the product, feature, or website you may look
+             for, I have a passion for software development, please review some of
+             my works and experience! if you like what you see and have a project
+             you need coded do not hesitate to contact me.
+          </p>
+          <ul class="list-format font-color">
+            <li><a href="https://github.com/flov3rh" class="fa-brands fa-square-github"></a></li>
+            <li><a href="https://www.linkedin.com/in/flov3rh/" class="fa-brands fa-linkedin"></a></li>
+            <li><a href="https://twitter.com/flov3rh" class="fa-brands fa-twitter"></a></li>
+            <li><a href="#" class="fa-solid fa-envelope"></a></li>
+          </ul>
+        <a href="#middle" id="down-button" class="fa-solid fa-chevron-down"></a>
+      </header>
```

Comment:
- Hey, the container of this content should be a `section` tag is this is the first section of your document to enforce a good use of the html5 semantic tag. please, kindly update this.